src/scripts/products.js

```diff
@@ -0,0 +1,64 @@
+(function () {
+    var productsCard = document.querySelectorAll(".js-products");
+    var jsDoots = document.getElementById("js-dots");

```

Comment:
jsDoots to pewnie literówka i miało być jsDots (popraw wszędzie gdzie tego używasz :)